api/click-house-client.ts

```diff
@@ -0,0 +1,64 @@
+import { createClient } from 'npm:@clickhouse/client'
+import {
+  CLICKHOUSE_HOST,
+  CLICKHOUSE_PASSWORD,
+  CLICKHOUSE_USER,
+} from './lib/env.ts'
+import { respond } from './lib/response.ts'
+import { log } from './lib/log.ts'
+import { ARR, NUM, OBJ, STR, UNION } from './lib/validator.ts'
+import { Asserted } from './lib/router.ts'
+
+const LogSchema = OBJ({
+  timestamp: STR(),
+  trace_id: STR(),
+  span_id: STR(),
+  severity_text: STR(),
+  severity_number: NUM(),
+  attributes: OBJ({}),
+  event_name: STR(),
```

Comment:
describe each fields, isn't `severity_text` & `severity_number` redundant ? do we ever use `severity_number` ?